packages/p2p-v2/src/components/PaymentMethodForm/PaymentMethodFormModalRenderer/PaymentMethodFormModalRenderer.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { PaymentMethodErrorModal, PaymentMethodModal } from '@/components/Modals';
+import { TFormState } from '@/reducers/types';
+import { TSocketError } from '@deriv/api-v2/types';
+
+type TPaymentMethodFormModalRendererProps = {
+    actionType: TFormState['actionType'];
+    createError: TSocketError<'p2p_advertiser_payment_methods'> | null;
+    isCreateSuccessful: boolean;
+    isModalOpen: boolean;
+    isUpdateSuccessful: boolean;
+    onResetFormState: () => void;
+    setIsModalOpen: (isModalOpen: boolean) => void;
+    updateError: TSocketError<'p2p_advertiser_payment_methods'> | null;
+};
+
+const PaymentMethodFormModalRenderer = ({
```

Comment:
Also, this stuff will be dealt away with when `modalManager` comes in right?

Reply:
yeah just for now i separated it cause it was getting quite cluttered in the form, and it can be dealt separately